docs/spec.md

```diff
@@ -6268,6 +6294,19 @@ If `x` is a value or placeholder, this function is a shortcut for
 `member_name(type(x))`.  If `x` is not a type that has an appropriate member, or
 a value or a placeholder of such a type, returns `None`.
 
+* `is_convertible_or_quantizable(x: Type, y: Type) -> bool` checks for the
+equality of `x` and `y`, ignoring the bitwidth, when they are of type
+`TensorElementType`. When `x` and `y` are `QuantizedTensorElementType`s,
+the function checks for the equality of `QuantizationExpressedType` component.
+
+```python
+def is_convertible_or_quantizable(x: Type, y: Type) -> Value:
+  return is_integer(x) = is_integer(y) or
+    is_float(x) = is_float(y) or
+    is_complex(x) = is_complex(y) or
```

Comment:
Shouldn't we have 'and' instead of = here?

Reply:
That is right. Fixed.